share/lib/python/neuron/rxd/species.py

```diff
@@ -1013,10 +984,11 @@ def _delete(self):
             if self in _extracellular_diffusion_objects:
                 del _extracellular_diffusion_objects[self]
                 # remove the grid id
-                for sp in _extracellular_diffusion_objects:
-                    if hasattr(sp,'_grid_id') and sp._grid_id > self._grid_id:
```

Comment:
Any idea why the hasattr was there before? I can't imagine that being needed, but not sure why it was there.